src/main/java/net/dv8tion/jda/core/entities/Guild.java

```diff
@@ -886,6 +886,102 @@ default Emote getEmoteById(long id)
     @CheckReturnValue
     RestAction<List<Ban>> getBanList();
 
+    /**
+     * Gets an {@link net.dv8tion.jda.core.entities.Guild.Ban Ban} of the provided ID
+     * <br>If you wish to ban or unban a user, please {@link GuildController#ban(User, int) GuildController.ban(User, int)} or
+     * {@link GuildController#unban(User) GuildController.ban(User)}.
```

Comment:
`{@link GuildController#unban(User) GuildController.ban(User)}`

Reply:
also "please **use** ..."

Reply:
ah that was too late kant i gotta patch it. sorry